.github/CODEOWNERS

```diff
@@ -133,6 +133,7 @@
 
 
 # BEGIN - pending clarification
+/aml/**/*                      @GoogleCloudPlatform/python-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
```

Comment:
this should include a team that is responsible for these samples.

Reply:
If possible, please create a github group that will take care of the samples in this directory.

Reply:
Done